### ISSUE TITLE
RDF and Property Graphs (reopend)

### ISSUE THREAD

**Issue by mhedenus** [w3c/EasierRDF#83]
Hello,

I am creating this issue although there are already issues covering this topic because it seams that discussion has been ceased there.

Short introduction:

I am working at Vitesco Technologies (formerly Continental Automotive, Regensburg, Germany) and my project is all about data integration. We are using RDF's data integration capabilities for integrating distributed and heterogenous data from all over the company. This effort is quite successful but it turned out that RDF is not intelligible for the users and even not for engineers. What almost everybody understands is Property Graph.

So we developed a new graph that is basically a "dual" graph:  it is Property Graph that is mapped to RDF.  We named it Information Graph. This under development for over 3 years now. We would be happy to share our experience. 

Is this the right place to bring this up ?

**Comment by draggett** [comment]
Do you have a pointer to your work?

**Comment by mhedenus** [comment]
No, it is currently not available in the public, but we think its mature enough to present it. I am looking for an opportunity to engage into disscusions with experts. When I found the report on the W3C Workshop on Web Standardization for Graph Data, I thought: hey, that's excatly what we're doing!

To outline it:

- We developed a Graph Model (and Java API) called Information Graph that is a Property Graph but we use the terminology of the Entity-Relationship model, i.e. we use Entities, Relations and Properties and their corresponding types
- Everything is identified using URIs
- Information Graph is serialized as RDF and persisted in RDF databases
- Queries are done with SPARQL
- Deserializing from RDF may need a model that specifies whether a RDF predicate is interpreted as a Relation or a Property
- Application logic is implemented also only using the ER model terms
- Communication towards users is only in terms of ER-model

Long story short:

Our experience is that RDF excellent for solving technical issues, but it is not suitable at all as mindset for modelling or application development. On the other hand Property Graph == ER-modelling is an excellent general frame / mindset / language for modelling.

Imagine a generic graph model that hides all the technical needs, makes it easy to use it for domain modelling but at same point of time is compatible with RDF.

**Comment by dbooth-boston** [comment]
> I am creating this issue although there are already issues covering this topic because it seams that discussion has been ceased there.

I think it would be better to continue on those discussion threads rather than opening a new issue, to avoid fragmenting the discussion.  It is fine to add new information to an old thread.  

> So we developed a new graph that is basically a "dual" graph: it is Property Graph that is mapped to RDF.

Interesting.  From your description it also sounds related to the [RDF-star](https://w3c.github.io/rdf-star/) work that's been going on for a while.  You should look at that if you have not already seen it.  But from your description, it sounds like one advantage of your approach is that it has a clear mapping to standard RDF.

> Imagine a generic graph model that hides all the technical needs, makes it easy to use it for domain modelling but at same point of time is compatible with RDF.

That sounds good, but we really need more information about it, to discuss it more fully -- at least a document that describes it in detail, but preferably also a github repo (under an [OSI-approved license](https://opensource.org/licenses)) that would allow people to try it.

Is your intent to keep this work proprietary or contribute it to the community?  Bear in mind that contributions to the EasierRDF effort are governed by W3C's [IP policy for community groups](https://www.w3.org/community/about/process/summary/), in order to keep them unencumbered.

**Comment by mhedenus** [comment]
Ok, I move then to issue https://github.com/w3c/EasierRDF/issues/45 and continue there.

Concerning the IP issues: speaking for myself I would publish our stuff as open source, but that is not the style of German industry.
